Week3/homework/step2-2.js

```diff
@@ -2,19 +2,29 @@
 
 function threeFive(startIndex, stopIndex, threeCallback, fiveCallback) {
   const numbers = [];
-
-  // Replace this comment and the next line with your code
-  console.log(startIndex, stopIndex, threeCallback, fiveCallback, numbers);
+  for (let i = startIndex; i <= stopIndex; i++) {
```

Comment:
Great that you did this all in one loop. Not sure why you're creating the `numbers` array and pushing into it though? It doesn't look like you're using this array anywhere?

Reply:
yes. we do not need it